lib/ruby_saml/settings.rb

```diff
@@ -296,8 +292,40 @@ def get_binding(value)
       }.freeze
     }.freeze
 
+    # @deprecated Will be removed in v2.1.0
+    def compress_request
+      compress_deprecation('compress_request', 'idp_sso_service_binding')
+      defined?(@compress_request) ? @compress_request : true
```

Comment:
One more thing I caught, the previous default for these was "true", so it needs to stay that way.